ui/src/components/CompletedTasksTable.tsx

```diff
@@ -20,6 +20,7 @@ import Alert from "@material-ui/lab/Alert";
 import AlertTitle from "@material-ui/lab/AlertTitle";
 import SyntaxHighlighter from "./SyntaxHighlighter";
 import { AppState } from "../store";
+import FileCopyOutlinedIcon from '@material-ui/icons/FileCopyOutlined';
```

Comment:
nit: Can you place this import so that it's under other `@material-ui/xxx` imports?

Reply:
done!